cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,553 @@
+General information on rule format
+==================================
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
+
+File Structure
+--------------
+
+Enrichment rules are defined with YAML format. The file extension has to be ".yml". Indentation is mandatory within the
+file and has to strictly follow the YAML format.
+
+This file starts with "rules" roots element, which contains the list of enrichment rules, of which this document is
+about. Each rules is referred by a code and can contain a list of conditions or actions. Actions are executed if all
+conditions are verified or if no action is defined.
```

Comment:
are met.
"Actions are executed [...] or if no action is defined" seems strange to me :/